extracted-files/tags/AFN

```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+type: legacy
+
+uri: https://gedcom.io/terms/v7/AFN
+
+descriptions:
+  - Ancestral File Number
+  - This URI is defined to support conversion from 5.5.1 to 7.0 and beyond.
+    It is intended for use as the https://gedcom.io/terms/v7/TYPE of an 
+    https://gedcom.io/terms/v7/EXID replacing 5.5.1's AFN structure.
+    
+    Ancestral File is a computerized collection of genealogies that links 
+    families into pedigrees, showing ancestors and descendants. An Ancestral
+    File Number was assigned to every record that was published in Ancestral
+    File.
+    
+    See <https://www.familysearch.org/wiki/en/Ancestral_File> for more.
```

Comment:
Why does line 18 have `<>` around the URL but lines 10 and 11 don't?   Whatever syntax is needed by YAML should be consistently used.

Reply:
Good call. I've fixed this and validated the files as YAML